docs/controllers.md

```diff
@@ -39,32 +39,32 @@ type, which allows users to specify:
 2. Protocol-specific information
 3. Passthrough metadata to send along with the to the cross-chain transfer
 
-Protocol-specific information is encoded in the `Attributes` field using an `any`
-type that implements the `ForwardingAttributes` interface.
-For implementation details, refer to the
+Protocol-specific information is encoded in the `Attributes` field using an `any` type that
+implements the `ForwardingAttributes` interface. For implementation details, refer to the
 [proto definition](https://github.com/noble-assets/orbiter/blob/main/proto/noble/orbiter/core/v1/orbiter.proto#L35-L55).
 
-Forwarding controllers interpret protocol-specific information and forward funds.
-Based on the selected bridge, the following controllers are available:
+Forwarding controllers interpret protocol-specific information and forward funds. Based on the
+selected bridge, the following controllers are available:
 
 - **AutoCCTP**: Automatic forwarding via the Cross-Chain Transfer Protocol
 - **AutoLane**: Automatic forwarding via the Hyperlane protocol
 - **AutoIBC**: Automatic forwarding via the Inter-Blockchain Communication (IBC) protocol
 
 ## CCTP
 
-The Cross-Chain Transfer Protocol (CCTP) is an interoperability protocol developed by Circle
-to enable native USDC transfers across different blockchain networks.
-CCTP uses a burn-and-mint mechanism where USDC is burned on the source chain and minted on the destination chain,
-ensuring that the total supply remains constant across all supported chains.
+The Cross-Chain Transfer Protocol (CCTP) is an interoperability protocol developed by Circle to
+enable native USDC transfers across different blockchain networks. CCTP uses a burn-and-mint
+mechanism where USDC is burned on the source chain and minted on the destination chain, ensuring
+that the total supply remains constant across all supported chains.
```

Comment:
seems like the formatter has removed the semantic line breaks here :eyes: or did you make these changes manually?